src/corehost/cli/ijwhost/ijwthunk.cpp

```diff
@@ -183,6 +183,7 @@ bool are_thunks_installed_for_module(pal::dll_t instance)
         {
             return true;
         }
+        currentChunk = currentChunk->GetNext();
```

Comment:
This is a bit unsettling. How was this missed during development? Was this path never exercised?

Reply:
This path is only hit when loading 2 or more IJW modules into the same image. I had only tested loading one IJW module into a DLL at a time since the only way I could get the initial testing at the time was with manually hex-editing the binary to refer to the new host instead of the old host.